compiler/rustc_codegen_llvm/src/debuginfo/create_scope_map.rs

```diff
@@ -84,7 +84,6 @@ fn make_mir_scope<'ll, 'tcx>(
     }
 
     let loc = cx.lookup_debug_loc(scope_data.span.lo());
```

Comment:
Can this binding be moved into the `None` arm too?

Reply:
Afraid not, it's used down on line 129 & 130